.gitignore

```diff
@@ -3,8 +3,14 @@
 .settings
 .clang-format
 src/fastsynth/fastsynth
+tasks.json
+*.code-workspace
 *.d
+*.idb
+*.ilk
 *.o
+*.obj
 *.out
+*.pdb
 *.log
 *.exe
```

Comment:
changed the git ignore?

Reply:
`*.pdb` are debug information files produced when creating a debug build on MSVC. Since we claim to support MSVC (and I am currently compiling on it 🙂) I'd prefer to keep those in. Also, since we already `.gitignore` Eclipse IDE file (`.project`) I thought adding the VSCode ones won't hurt (which again I am using).